search.html

```diff
@@ -0,0 +1,263 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
+<!--
+%CopyrightBegin%
+
+Copyright Ericsson AB 2009-2020. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+%CopyrightEnd%
+-->
+<html>
+
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1"></meta>
+	<meta charset="utf-8"></meta>
+	<link rel="stylesheet" href="doc/otp_doc.css" type="text/css" />
+	<title>Erlang/OTP doc search</title>
+	<script id="js" type="text/javascript" language="JavaScript" src="doc/js/flipmenu/flipmenu.js">
+	</script>
+	<script id="js2" type="text/javascript" language="JavaScript" src="doc/js/topbar.js">
+	</script>
+	<script src="https://cdn.jsdelivr.net/npm/algoliasearch@4.5.1/dist/algoliasearch-lite.umd.js" integrity="sha256-EXPXz4W6pQgfYY3yTpnDa3OH8/EPn16ciVsPQ/ypsjk=" crossorigin="anonymous"></script>
+	<script src="https://cdn.jsdelivr.net/npm/instantsearch.js@4.8.3/dist/instantsearch.production.min.js" integrity="sha256-LAGhRRdtVoD6RLo2qDQsU2mp+XVSciKRC8XPOBWmofM=" crossorigin="anonymous"></script>
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/instantsearch.css@7.4.5/themes/satellite-min.css" integrity="sha256-TehzF/2QvNKhGQrrNpoOb2Ck4iGZ1J/DI4pkd2oUsBc=" crossorigin="anonymous"> </head>
+
+<style>
```

Comment:
Would be nice get this applied across the boards for /docs but I honestly exactly wasn't sure how to do that. There's a doc.scss but that doesn't seem to generate `_site/doc/otp_doc.css` which is an artifact. I think it's something that could be done in another PR too.

Reply:
otp_doc.css is taken from https://github.com/erlang/otp/blob/master/lib/erl_docgen/priv/css/otp_doc.css and then massaged using [otp_flatten_docs](https://github.com/erlang/erlang-org/blob/master/_scripts/otp_flatten_docs) to get the final result.

The relevant parts to update would be [at the bottom](https://github.com/erlang/erlang-org/blob/master/_scripts/otp_flatten_docs#L423-L453) of that perl script.

I'm not sure I understand exactly what you want to achieve though?

Reply:
Ahh, so I should have been more specific : 

```
.DocSearch-Button { 
    margin: 0 0 0 0px !important; 
}
```

That's the main thing that i think needs to get applied to all pages. It centers other elements, besides that it's just to avoid having styles defined in the page itself. Not a huge deal.

Reply:
Note: I tried updating the sub-routine in the perl script, but doesn't seem to output, there's a condition I'm missing. I think this can be done in another PR unless you know why that would be off the top of your head. I haven't dug into how everything is constructed yet FWIW.

Reply:
No worries, I can fix it now that I know what you are after.